src/bootstrap/bootstrap.py

```diff
@@ -1373,11 +1373,26 @@ def main():
         sys.argv[1] = "-h"
 
     args = parse_args(sys.argv)
-    help_triggered = args.help or len(sys.argv) == 1
 
-    # If the user is asking for help, let them know that the whole download-and-build
+    # Root help (e.g., x.py --help) prints help from the saved file to save the time
+    if len(sys.argv) == 1 or sys.argv[1] in ["-h", "--help"]:
```

Comment:
The second part of this seems like it's not quite right, it needs `(sys.argv[1] in ["-h", "--help"] and len(sys.argv) == 2)`, right? Otherwise `x.py --help check` wouldn't do the right thing?

Reply:
> Or, do you think~

I guess if we dispatch `x.py --help check` to bootstrap binary. Clap returns the top level help for it unless we modify the behaviour.